### PR TITLE
OCPBUGS-16363: Added a procedure for Hypershift management plane conf…

### PR DESCRIPTION
**Pull request by sheriff-rh**
…iguration

Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OCPBUGS-16363

Link to docs preview (VPN required):
[Configuring the Hosted control planes management cluster](https://file.rdu.redhat.com/antaylor/OCPBUGS-16363/security/compliance_operator/compliance-scans.html#co-hcp-mgmt-config_compliance-operator-scans)

QE review:
- [x] QE has approved this change.

Additional information:
None

### CODE REVIEW

File: modules/compliance-operator-hcp-mgmt-config.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-installation.adoc
+
+:_content-type: PROCEDURE
+[id="co-hcp-mgmt-config_{context}"]
+= Configuring the Hosted control planes management cluster
+
+If you are hosting your own Hosted control plane or Hypershift environment, you will need to set the prefix for the namespaces in the Hosted control plane. You can achieve this by creating a `TailoredProfile`.
```
Comment (yuumasato) [inline modules/compliance-operator-hcp-mgmt-config.adoc:9]:
```suggestion
If you are hosting your own Hosted control plane or Hypershift environment and want to scan a Hosted Cluster, you will need to set the name and prefix namespace for the target Hosted Cluster. You can achieve this by creating a `TailoredProfile`.
```

File: modules/compliance-operator-hcp-mgmt-config.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE       NAME                   VERSION   KUBECONFIG                              PROGRESS    AVAILABLE   PROGRESSING   MESSAGE
+local-cluster   79136a1bdb84b3c13217   4.13.5    79136a1bdb84b3c13217-admin-kubeconfig   Completed   True        False         The hosted control plane is available
+----
+
+. In the management cluster, write a `TailoredProfile` to define the namespaces in the Hosted control plane:
```
Comment (yuumasato) [inline modules/compliance-operator-hcp-mgmt-config.adoc:36]:
```suggestion
. In the management cluster, create a `TailoredProfile` extending the Profile you want to scan with, and define the name and namespace of the Hosted Cluster to be scanned.
```

**Review by yuumasato — COMMENTED** [review]
Hey @sheriff-rh,

I suggested some clarifications, feel free to change and rephrase them as you like.

**Comment by sheriff-rh** [comment]
All feedback included in the latest update.

File: modules/compliance-operator-hcp-mgmt-config.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+This procedure only applies to users managing their own Hosted control planes environment.
+====
+
+.Prerequisites
+
+* The Compliance Operator is installed in the management cluster.
+
+.Procedure
+
+. Obtain the `name` and `namespace` of the management cluster by running the following command:
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply modules/compliance-operator-hcp-mgmt-config.adoc]: Yes, I think we should do that

File: modules/compliance-operator-hcp-mgmt-config.adoc [hunk 4]
```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: PROCEDURE
+[id="co-hcp-mgmt-config_{context}"]
+= Configuring the Hosted control planes management cluster
+
+If you are hosting your own Hosted control plane or Hypershift environment and want to scan a Hosted Cluster, you will need to set the name and prefix namespace for the target Hosted Cluster. You can achieve this by creating a `TailoredProfile`.
```
Comment (Vincent056) [inline modules/compliance-operator-hcp-mgmt-config.adoc:9]: Do you think it is better if we say:
```
If you are hosting your own Hosted control plane or Hypershift environment and want to scan a Hosted Cluster from the management cluster.```

Comment (sheriff-rh) [reply modules/compliance-operator-hcp-mgmt-config.adoc:9]: Yep, I like it. I'll make the update.

**Comment by sheriff-rh** [comment]
Latest feedback included in this update.

File: modules/compliance-operator-hcp-mgmt-config.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply modules/compliance-operator-hcp-mgmt-config.adoc]: What is the `ocp4-ics` profile? I don't see anything that matches that description the [supported profiles](https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html) page.

[the remark this answers is not in the document]
Comment (sheriff-rh) [reply modules/compliance-operator-hcp-mgmt-config.adoc]: @Vincent056 clarified this is `ocp4-cis`. I will add a note for supporting only these profiles.

**Comment by sheriff-rh** [comment]
Latest feedback included in this update.

File: modules/compliance-operator-hcp-mgmt-config.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (xiaojiey) [reply modules/compliance-operator-hcp-mgmt-config.adoc]: sorry, typo error.

**Comment by sheriff-rh** [comment]
@xiaojiey can you review one more time when you get a chance? Thanks!

File: modules/compliance-operator-hcp-mgmt-config.adoc [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: hypershift-cisk57aw88gry
+  namespace: openshift-compliance
+spec:
+  description: This profile test required rules
+  extends: ocp4-cis
```
Comment (xiaojiey) [inline modules/compliance-operator-hcp-mgmt-config.adoc:53]: Generally it is look good. To make it more clear, maybe we can make the extends as variable, and clarify only `ocp4-cis` and `ocp4-pci-dss` is supported. If so, we can remove the previous NOTE.

Comment (sheriff-rh) [reply modules/compliance-operator-hcp-mgmt-config.adoc]: That's a great idea. I updated the callouts: 
image

I also left the NOTE at the top to reinforce the knowledge.

**Comment by sheriff-rh** [comment]
Merging!

**Pull request merged**